wpimath/src/main/native/include/frc/estimator/UnscentedTransform.h

```diff
@@ -61,14 +66,20 @@ SquareRootUnscentedTransform(
   }
   Sbar.template block<CovDim, CovDim>(0, States * 2) = squareRootR;
 
-  // Merwe defines the QR decomposition as Aᵀ = QR
+  // Compute the square-root covariance of the sigma points
+  // This is upper triangular, so we need to take the transpose
+  // equations (20) and (24)
```

Comment:
The lack of a newline before this implies it's a continuation of the previous comment, which doesn't make grammatical sense. There's some more cases of this elsewhere.